flutter_local_notifications/README.md

```diff
@@ -144,7 +315,7 @@ var result = await flutterLocalNotificationsPlugin
 
 Here the call to `flutterLocalNotificationsPlugin.resolvePlatformSpecificImplementation<IOSFlutterLocalNotificationsPlugin>()` returns the iOS implementation of the plugin that contains APIs specific to iOS if the application is running on iOS. The `?.` operator is used here as the result will be null when run on other platforms. Developers may alternatively choose to guard this call by checking the platform their application is running on.
 
-### Displaying a notification
+### Display a notification
```

Comment:
What was the reasoning for the wording change here? Currently it's inconsistent with the previous one that is trying to communicate "This section is about requesting notification permissions". I'm guessing the thought process here is that "this section is on how to display a notification"?

I know there was some inconsistency in that wording choice before having taken a closer look. If this is going to be changed then I would suggest it be done across the board while we're at it :)

Reply:
All good no need to explain, it has been fixed.